src/System.Private.CoreLib/shared/System/Globalization/CultureData.cs

```diff
@@ -812,27 +802,15 @@ internal string CultureName
             }
         }
 
-        // Are overrides enabled?
-        internal bool UseUserOverride
-        {
-            get
-            {
-                return _bUseOverrides;
-            }
-        }
+        /// <summary>
+        /// Are overrides enabled?
+        /// </summary>
+        internal bool UseUserOverride => _bUseOverrides;
 
-        // locale name (ie: de-DE, NO sort information)
-        internal string SNAME
```

Comment:
>SNAME [](start = 24, length = 5)

I am ok to change the names here but these was named according to Windows LCID type names which was easily from the name of the properties to know what we are calling from Windows. 

As I mentioned I am ok because we support non Windows platforms now.

Reply:
Yeah it makes sense to name them appropriately that makes sense - I've updated all the remaining occurences